src/util/apis/telemetry-filter.ts

```diff
@@ -15,11 +15,12 @@ const sessionHeaderName = "diagnostic-context";
 const commandNameHeaderName = "cli-command-name";
 
 export function telemetryFilter(commandName: string, telemetryIsEnabled: boolean) : {(resource: WebResource, next: any, callback: any): any} {
+  const telemetrySource = process.env.TELEMETRY_SOURCE || "cli";

```

Comment:
Please add a function to the src/util/profile/environment-vars.ts file to read this environment variable. That way all the evenironment variables we look for are in a single place in the code.

Also prefix the variable name with APPCENTER: `APPCENTER_TELEMETRY_SOURCE`.